### PR TITLE
doc: Wrote a changelog

### PR DESCRIPTION
**Pull request by de-abreu**
A proposal to close [133](https://github.com/courselab/coral/issues/133).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -7,20 +7,169 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+Changed
+Deprecated
+Removed
+Fixed
+Security
+
+MAJOR.MINOR.PATCH
+    MAJOR version when you make incompatible API changes
+    MINOR version when you add functionality in a backward compatible manner
+    PATCH version when you make backward compatible bug fixes
```
Comment (lincolncpp) [inline ChangeLog.md:175]: is this some kind of instruction for the changelog? if so, should we move these lines to `CONTRIBUTING.md` file instead?

**Comment by monacofj** [comment]
Well, according to our workflow, we haven't yet made any public release.
I was expecting `ChangeLog.m` to have two entries: 

- `0.1.0` that is the initial state since the project was created
- `1.0.0' that will be the first release by Dec 17 2024

The intermediate steps are development upgrades and do not consist in releases (there are not tags for them in the repo).

If we are going to stick with conventional changelog, all changes, fixes etc. after the initial code will comprise the v1.0.0 release.

File: ChangeLog.md (hunk 1 above)
Comment (de-abreu) [reply ChangeLog.md:175]: That... is a mistake.

**Comment by de-abreu** [comment]
I see. That should make it much simpler. I'll commit a new version soon.

**Comment by rodrigo-lex** [comment]
lgtm! However, I would also add to the "Added" section the addition of the orange (#96).

**Comment by monacofj** [comment]
Good job.
But I was wondering. There were a few bugs in KobraPy and it's ok list their fixes under section `Fix`.
All other fixes are fixes for new features that did not exist in KobraPy. I believe that listing them as fixes of KobraPy code does not make much sense. Against this background, think v1.0.0 should have only added, removed or changed things...

**Pull request merged**